library/src/ui/index.js

```diff
@@ -85,4 +86,107 @@ window.CoNDeT.ui = {
       this.removeChild(childToRemove);
     },
   },
+
+  DisplayComponent: (function () {
+    function constructor() {
+      common = {};
+    }
+
+    constructor.prototype.setCurrentXY = function ({ x, y }) {
+      common.deltaXY = { x: x, y: y };
+    };
+
+    constructor.prototype.getCurrentXY = function () {
+      return common.deltaXY;
+    };
+
+    constructor.prototype.prepareData = function (tablesJSON) {
+      var tablesList = [];
+      for (var i = 0; i < tablesJSON.length; i++) {
+        var tableProps = tablesJSON[i];
+        var rowsPrepared = [];
+        for (var j = 0, k = 0; j < tableProps.columns.conditions.length; j++) {
+          if (tableProps.rows.conditions[k][1] != null) {
+            rowsPrepared.push(tableProps.rows.conditions[k][1]);
+            k += 1;
+          } else {
+            rowsPrepared.push("");
+          }
+        }
+        for (var j = 0, k = 0; j < tableProps.columns.decisions.length; j++) {
+          if (tableProps.rows.decisions[k][1] != null) {
+            rowsPrepared.push(tableProps.rows.decisions[k][1]);
+            k += 1;
+          } else {
+            rowsPrepared.push("");
+          }
+        }
+
+        tablesList.push({
+          name: tableProps.name,
+          class: tableProps.classType,
+          coordinates: tableProps.coordinates,
+          conditions: tableProps.columns.conditions,
+          decisions: tableProps.columns.decisions,
+          rows: rowsPrepared,
+        });
+      }
+
+      return tablesList;
+    };
+
+    constructor.prototype.onInit = function (common, props) {
+      var ref = props.className;
+      var tables = this.prepareData(props.state.data);
+      for (var i = 0; i < tables.length; i++) {
+        this.createChild(window.CoNDeT.ui.Table, tables[i]);
+      }
+    };
```

Comment:
tutaj ref to będzie element w dom już utworzony o klasie `props.className`. Trzeba go pobrać z DOM i zwrócić